pages/arduino-info.mdx

```diff
@@ -0,0 +1,129 @@
+import Head from 'next/head'
+import Meta from '@hackclub/meta'
+
+<Meta
+  as={Head}
+  name="Summer of Making"
+  title="About your Arduino Starter Kit"
```

Comment:
Adding a description will improve seo & open graph previews

Reply:
not really caring much about seo, the only way any one really needs to find this site is through giving a link, its for after people get a kit and are waiting